pipeline/scripts/common_scenarios.jl

```diff
@@ -0,0 +1,41 @@
+using DrWatson
+@quickactivate "Analysis pipeline"
+
+include(srcdir("AnalysisPipeline.jl"))
+include(scriptsdir("common_param_values.jl"))
```

Comment:
i.e here this means we have random stuff floating in our global environment some of which is used and some of which is not which is not great practice?